src/spec/client/services/product-projections.spec.coffee

```diff
@@ -98,10 +110,11 @@ describe 'ProductProjectionService', ->
       .filter('foo:bar')
       .filterByQuery('foo:bar')
       .filterByFacets('foo:bar')
+      .markMatchingVariants(true)
```

Comment:
Query for the test above does not use search [endpoint](https://github.com/sphereio/sphere-node-sdk/blob/ae72a3718dfda20adb45c91407eba7a7b653fea1/src/coffee/services/product-projections.coffee#L306) but markMatchingVariants is exclusive feature of search which is not available over product-projections endpoint.